### ISSUE TITLE
Many action metadata file missing description - prevents changing defaults an re-registering them

### ISSUE THREAD

**Issue by punkrokk** [StackStorm-Exchange/stackstorm-aws#76]
When trying to add a default value to the ec2_create_tag action, I discovered many of the metadata files are missing descriptions. The outcome for me is I get a weird error trying to reregister, that I had to troubleshoot. 

Not sure if there is a desire to fix that or if simply there is a build process that should populate them is broken. Happy to help if needed.

**Comment by arm4b** [comment]
Yes, it's probably autogeneration process that doesn't populate descriptions properly.

Improvements in this area would be great to see :+1:

**Comment by punkrokk** [comment]
Where can I see the autogen code? 

Cheers, 

JP

**Comment by LindsayHill** [comment]
https://github.com/StackStorm-Exchange/stackstorm-aws/tree/master/etc/st2packgen
